src/frontend/get-github-setup.ts

```diff
@@ -5,14 +5,24 @@ import { Installation } from "../models";
 /*
 	Handles redirects for both the installation flow from Jira and
 	the installation flow from GH.
-	- From Jira: user has already installed the app and is redirected to the connect an org pg
+	- From Jira: user has already installed but wants to connect to an org not listed. Ater selecting an org in GH and giving repo permissions:
+		- they use the jiraHost we have in the cookie, connect, and are redirected to the GH config page
+		- they choose to enter an alternate jiraHost and are redirected to marketplace
 	- From GH:
-			- If we have the users Jira host, redirect to marketplace.
-			- Otherwise, render the setup page.
+		- If we have the users Jira host, redirect to marketplace.
+		- Otherwise, render the setup page (POST).
 */
 export default async (req: Request, res: Response): Promise<void> => {
 	req.log.info("Received get github setup page request");
-	const { jiraHost } = res.locals;
+	const { jiraHost, github, client } = res.locals;
+	const { data: { installations } } = await github.apps.listInstallationsForAuthenticatedUser();
```

Comment:
Looked into using our GH client but it needs to be updated to support user tokens. I've created a ticket in the backlog for us to address this and to update this afterwards.